### ISSUE TITLE
Other Lists and Preface must be placed between List of Figures and the undoing of the chapter indent

### ISSUE THREAD

**Issue by piyanatk** [GarenSidonius/ASU-Dissertation-Template#15]
Per ASU formatting guideline, other lists must immediately follow list of figures. Preface must follow all the lists. They must have roman page numbers and appear in the "CHAPTER" level in the table of contents. To do this you have to place other lists and preface between the list of figure and the undoing of chapter indent in the frontmatter. Please see the attached PDF for the output.

```latex
\clearpage
\pagestyle{ASUlof}
\listoffigures                          % List of Figures should appear in TOC, so use
                                        %   unstarred version of \listoffigures

% Other lists and preface go between here -->
\chapter{Definitions}                   %~OTHER LISTS (optional)
\input{definitions}                     %<Enter the name of the .tex file or omit this
                                        %   line and type in here.

\chapter{Preface}                       %~PREFACE (optional, less than 10 pages)
\input{sample_preface}                  %<Enter the name of the .tex file or omit this
                                        %   line and type in here.
                                        % Note: sample_preface contains \lipsum[66]
% <-- and here.

\phantomsection                         % \phantomsection is needed before using
                                        %   \addtocontents when it contains certain macros
                                        %   when also using 'hyperref' package
\addtocontents{toc}%                    % Undo manual override above for chapter indent,
  {\setlength{\cftchapterindent}%       %   so actual chapters in the TOC are indented
    {\levelindentincrement}%            %   correctly
  }
```
[dissertation_template_latex_sample.pdf](https://github.com/GarenSidonius/ASU-Dissertation-Template/files/3056542/dissertation_template_latex_sample.pdf)

**Comment by piyanatk** [comment]
Correct. The preface and other lists just have to be moved there.

**Comment by GarenSidonius** [comment]
This is resolved: 8361bac9f6a0f20cab773865d3f1b1672958cfd2
